lib/debezium/converters/basic.go

```diff
@@ -40,3 +41,19 @@ func (Int64Passthrough) Convert(value any) (any, error) {
 
 	return value, nil
 }
+
+type Bit struct{}
```

Comment:
Could call this `Base64` since it's a converter that produces base64 encoded values.

Reply:
sgtm